docs/components/encoder/_index.md

```diff
@@ -44,6 +44,210 @@ For configuration information, click on one of the following models:
+
+```python
+from viam.components.encoder import Encoder
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```go
+import (
+  "go.viam.com/rdk/components/arm"
+)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## API
+
+The encoder component supports the following methods:
+
+Method Name | Description
+----------- | -----------
+[GetPosition](#getposition) | Get the current position of the encoder.
+[ResetPosition](#resetposition) | Reset the position to zero.
+[GetProperties](#getproperties) | Get the supported properties of this encoder.
+
+### GetPosition
+
+Get the current position of the encoder in ticks or degrees.
+Relative encoders return ticks since last zeroing.
+Absolute encoders return degrees.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `position_type` (PositionType.ValueType): Specify whether you want the current position in `"ticks"` or in `"degrees"`.
+- `extra` [(Optional[Dict[str, Any]])](https://docs.python.org/library/typing.html#typing.Optional): Extra options to pass to the underlying RPC call.
+- `timeout` [(Optional[float])](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
+
+**Returns:**
+
+- (Tuple[[float](https://docs.python.org/3/library/functions.html#float), PositionType.ValueType]): The current position in ticks or degrees, and the type of position the encoder returns (ticks or degrees).
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/encoder/client/index.html#viam.components.encoder.client.EncoderClient.get_position).
+
+```python
+my_encoder = Encoder.from_robot(robot=robot, name='my_encoder')
+
+# Get the position of the encoder
+position = await my_encoder.get_position("ticks")
+print("The encoder position is currently ", position[0], position[1])
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `positionType` ([PositionType](https://pkg.go.dev/go.viam.com/rdk/components/encoder#PositionType)): Specify whether you want the current position in `"ticks"` or in `"degrees"`.
+- `extra` [(map[string]interface{})](https://pkg.go.dev/google.golang.org/protobuf/types/known/structpb): Extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- [(float64)](https://pkg.go.dev/builtin#float64): The current position in ticks or degrees.
+- [(PositionType)](https://pkg.go.dev/go.viam.com/rdk/components/encoder#PositionType): The type of position the encoder returns (ticks or degrees).
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/encoder#Encoder).
+
+```go
+myEncoder, err := encoder.FromRobot(robot, "my_encoder")
+if err != nil {
```

Comment:
You can remove logger lines, we don’t have equivalent in python and I believe it’s shown in code samples now 

This is something James suggested doing in the past (removing logging for brief samples unless it was directly applicable)

Reply:
Now that we've taken out the longer examples at the top, where would we put that?

Reply:
I guess if we include the err in the :=, we don't need the `if` bit because people using the Go SDK will need to know about or learn about error handling elsewhere anyway

Reply:
keep it here for now and we'll sort it out after merge - since this is somewhat urgent to land. My proposal is to have the package import ant then this piece of code once in the connect section